tools/mtmd/mtmd-helper.h

```diff
@@ -31,6 +32,7 @@ MTMD_API mtmd_bitmap * mtmd_helper_bitmap_init_from_file(mtmd_context * ctx, con
 //     image: formats supported by stb_image: jpg, png, bmp, gif, etc.
 //     audio: formats supported by miniaudio: wav, mp3, flac
 // note: audio files will be auto-detected based on magic bytes
+// note (2): this function does NOT support video input yet
 // returns nullptr on failure
 // this function is thread-safe
 MTMD_API mtmd_bitmap * mtmd_helper_bitmap_init_from_buf(mtmd_context * ctx, const unsigned char * buf, size_t len);
```

Comment:
Note: support for loading video from `mtmd_helper_bitmap_init_from_buf` will be added later on, in another PR